modules/cnv-about-hostpath-provisioner.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-configuring-local-storage-for-vms.adoc
+
+[id="cnv-about-hostpath-provisioner_{context}"]
+= About the hostpath provisioner
+
+The hostpath provisioner is a local storage provisioner designed for
+{CNVProductName}. If you want to configure local storage for
+virtual machines, you must enable the hostpath provisioner first.
+
+When you install the {CNVProductName} Operator, the hostpath provisioner Operator
+is automatically installed. To use it, you must:
+
+* Configure SELinux:
+** If you use Red Hat Enterprise Linux CoreOS 8 workers, you must create a MachineConfig
+object on each node.
+** Otherwise, apply the SELinux label `container_file_t` to the PersistentVolume (PV) backing
+directory on each node.
+* Create a HostPathProvisioner custom resource.
+* Create a StorageClass object for the provisioner in the `openshift-cnv` namespace.
```

Comment:
'StorageClass' object is cluster scoped, it doesn't need namespace.

Reply:
Fixed.